packages/shared/src/modules/groups/libs/validation-schemas/group-create.validation-schema.ts

```diff
@@ -0,0 +1,22 @@
+import { z } from "zod";
+
+import { GroupValidationMessage, GroupValidationRule } from "../enums/enums.js";
+import { type GroupCreateRequestDto } from "../types/types.js";
+
+const groupCreate: z.ZodType<GroupCreateRequestDto> = z.object({
+	name: z
+		.string()
+		.trim()
+		.min(GroupValidationRule.NAME_MINIMUM_LENGTH, {
+			message: GroupValidationMessage.NAME_REQUIRED,
+		})
+		.max(GroupValidationRule.NAME_MAXIMUM_LENGTH, {
+			message: GroupValidationMessage.NAME_TOO_LONG,
+		}),
+	permissionIds: z.array(z.number().int().positive()).optional(),
```

Comment:
I would say it's not optional, it's required in payload, but the array itself can be empty